### PR TITLE
Fix answers questions count

### PR DESCRIPTION
**Pull request by andela-oakinniranye**

### CODE REVIEW

**Comment by andela-echigbo** [comment]
:+1:

**Comment by andela-ooranagwa** [comment]
:+1:

File: lib/tasks/es.rake [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      model_name.import force: true
+    end
+
+    desc "Creates a new version of the model_name on ES. Usage rake zhishi:es:revise[question]"
+    task :revise, [:model_name] => :environment do |_, arg|
+      model_name = arg[:model_name].singularize.camelize.constantize
+      model_name.import
+    end
+  end
+end
```
Comment (andela-echigbo) [inline lib/tasks/es.rake]: Does this file have to be named `es.rake`? Is it like a convention? :question:

Comment (andela-oakinniranye) [inline lib/tasks/es.rake]: No @andela-echigbo. It's just a bit of convention-ish stuff

**Pull request merged**